pages/testlibrary.tsx

```diff
@@ -21,21 +32,29 @@ const TestLibrary: NextPage<TestlibraryProps> = ({sessionCandidate}) => {
             <Head>
                 <title>TestSwitch Test Library</title>
             </Head>
-            <TestLibraryStepper key={key} candidateTestStatuses={sessionCandidate.testStatuses}/>
+            <TestLibraryStepper key={key} candidateTestStatuses={useStatus()} />
         </Layout>
     );
 };
 
 export const getServerSideProps: GetServerSideProps = async ({query, res}) => {
     await assertTokenIsValid(query, res);
     const token = query.token as string;
-    const sessionData = getSessionCandidate(token);
-
     return {
         props: {
-            sessionCandidate: await sessionData
+            sessionCandidate: await getSessionCandidate(token)
         }
     }
 };
 
-export default TestLibrary;
\ No newline at end of file
+export interface SessionCandidate {
+    firstName: string;
+    lastName: string;
+    testStatuses: CandidateTestStatus[];
+}
+export interface NewTestSubmission{
+    testId: number;
+    testAnswer: string;
+}
```

Comment:
I think I've seen these definitions 3 times - we probably only need 1 :)

Reply:
put in models (one version of each only!)